src/main/java/github/scarsz/discordsrv/DiscordSRV.java

```diff
@@ -1077,4 +1077,26 @@ public static boolean isUpdateCheckDisabled() {
                 !config().getBooleanElse("UpdateCheckDisabled", false);
     }
 
+    public static final String DEFAULT_ROLE_ID = "000000000000000000";
+    public static final Map<String, String> DEFAULT_GROUP_AND_ROLES_TO_SYNC = new HashMap<String, String>() {{
+        put("trusted", DEFAULT_ROLE_ID);
+        put("vip", DEFAULT_ROLE_ID);
+    }};
+
+    /**
+     * @return Whether or not DiscordSRV group role synchronization has been enabled in the configuration.
+     */
+    public static boolean isGroupRoleSynchronizationEnabled() {
+        final Map<String, String> groupsAndRolesToSync = DiscordSRV.config().getMap("GroupRoleSynchronizationGroupsAndRolesToSync");
+        if (groupsAndRolesToSync.isEmpty()) return false;
+        if (groupsAndRolesToSync.equals(DEFAULT_GROUP_AND_ROLES_TO_SYNC)) return false;
+        for(Map.Entry<String, String> entry : groupsAndRolesToSync.entrySet()) {
+            final String group = entry.getKey();
+            final String roleId = entry.getValue();
+            if (group.isEmpty()) return false;
+            if (roleId.isEmpty() || roleId.equals(DEFAULT_ROLE_ID)) return false;
```

Comment:
This returns false even when there is a valid id and one or more default values

Reply:
We have choices:
1. only look for default configuration and/or completely blank
2. look for any role being 18 x 0's
3. look for at least one role not being 18 x 0's
am guessing you'd prefer option 3?

Reply:
number 3 would be ideal here, yes

Reply:
How do you feel about the constant `DEFAULT_ROLE_ID` being declared? Can easily get rid of it and just put the string in instead.

Reply:
doesn't really make a difference, though using a constant is inconsistent with other checks in DiscordSRV